mp/src/game/server/momentum/mom_lobby_system.cpp

```diff
@@ -73,7 +73,7 @@ void CMomentumLobbySystem::HandleNewP2PRequest(SteamNetworkingMessagesSessionReq
     if (!IsInLobby(hUserID))
         return;
 
-    if (IsUserBlocked(hUserID))
+    if (m_vecBlocked.HasElement(hUserID))
```

Comment:
I think the old logic should be restored here as well and use `IsUserBlocked` to add the user immediately to the `m_vecBlocked` vector above this line.

Reply:
Oh right, we need to account for people who just opened the game and already had others blocked.